declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="customizing-permissions-by-creating-aggregated-cluster-roles"]
+= Customizing permissions by creating aggregated cluster roles
+:context: customizing-permissions-by-creating-aggregated-cluster-roles
+
+toc::[]
+
+The default cluster role for the Argo CD Application Controller has a specific set of hard-coded permissions. The {gitops-title} Operator manages this cluster role, so you cannot modify it. As a cluster administrator, you can customize the permissions by using any one of the following methods:
+
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc#customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances[Creating user-defined cluster roles]
+* xref:../declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc#gitops-creating-aggregated-cluster-roles_customizing-permissions-by-creating-aggregated-cluster-roles[Creating aggregated cluster roles]
+
+With aggregated cluster roles, you can aggregate several cluster roles into one combined cluster role. You can enable users to easily add user-defined permissions for the Argo CD Application Controller. You do not have to define permissions by creating a completely new cluster role from scratch.
+
+[IMPORTANT]
+====
+* You can create aggregated cluster roles only for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
+* Deleting the `aggregatedClusterRoles` field from the Argo CD custom resource (CR) does not delete the user-defined cluster role. You have to manually delete the user-defined cluster role using the CLI or UI for the efficient use of resources and memory.
+====
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You have the knowledge about link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles[aggregated cluster roles] and how they work.
+* You have installed {gitops-title} 1.14.0  or a later version on your {OCP} cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have installed a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance] in your defined namespace. For example, `spring-petclinic` namespace. 
+* You have validated that the user-defined cluster-scoped instance is configured with the cluster roles and cluster role bindings for the following components:
++
+** Argo CD Application Controller
+** Argo CD server
+** Argo CD ApplicationSet Controller (provided the ApplicationSet Controller is created)
```

Comment:
Suggestion: Can we explain the bracket part a bit better? I am not in favor of including it in brackets and want to merge it a single sentence.

Reply:
Sorted the conversation with @Dhruv-Soni11. Retaining the content as is.